synfig-studio/plugins/lottie-exporter/common/Param.py

```diff
@@ -799,6 +828,25 @@ def __get_value(self, frame):
                 else:
                     ret = math.sin(angle)*amp
 
+            elif self.param[0].tag == "cos":
+                angle = self.subparams["cos"].subparams["angle"].__get_value(frame)
+                amp = self.subparams["cos"].subparams["amp"].__get_value(frame)
+                angle = math.radians(angle)
+                if isinstance(amp, list):
+                    ret = [0, 0]
+                    ret[0] = math.cos(angle) * amp[0]
+                    ret[1] = math.cos(angle) * amp[1]
+                else:
+                    ret = math.cos(angle)*amp
+
+            elif self.param[0].tag == "fromint":
+                link = self.subparams["fromint"].subparams["link"].__get_value(frame)
+                if isinstance(link, list):
+                    ret = [0, 0]
+                    ret[0] = int(link[0]/60)*60.0
+                    ret[1] = int(link[1]/60)*60.0
+                else:
+                    ret = int(link/60)*60.0
```

Comment:
If you add the following line here:
`print(frame, ret)`
You will be able to see that the actual values produced by lottie differ from the values produced here.
I am attaching the `.sif` used by me
[integer_circle.zip](https://github.com/synfig/synfig/files/4348879/integer_circle.zip)

Also, instead of 60 here, I guess you need to use [settings.PIX_PER_UNIT](https://github.com/synfig/synfig/blob/77164e8371269afb0490287f200f6c35fe067e88/synfig-studio/plugins/lottie-exporter/common/misc.py#L33)

Reply:
You are getting a wrong value, mostly because `fromint` uses `round()` instead of `int()`.
Am I right here @ice0 ?

Reply:
Changed int with round and it produces the correct values. So I guess this is what needed to be done :smiley: .
Thank you for the help ^_^